test/unit/components/navbars.spec.js

```diff
@@ -21,15 +21,15 @@ describe('components', function() {
 
     describe('navbarAbsoluteTop', function() {
       it('adds class has-navbar-top if navbarAbsoluteTop is present', function() {
-        let elem = angular.element('<div class="navbar-absolute-top" />');
+        let elem = angular.element('<div class="navbar-absolute-top" nav-controller />');
```

Comment:
Ya I didn't think that it would be very useful either. I was just using the
other test as a guide.

On Sep 7, 2016 2:22 PM, "Maurizio Casimirri" <notifications@github.com>
wrote:

> In test/unit/components/navbars.spec.js
> <https://github.com/mcasimir/mobile-angular-ui/pull/378#discussion_r77906493>
> :
>
> > @@ -21,15 +21,15 @@ describe('components', function() {
> >
> >      describe('navbarAbsoluteTop', function() {
> >        it('adds class has-navbar-top if navbarAbsoluteTop is present', function() {
> > -        let elem = angular.element('<div class="navbar-absolute-top" />');
> > +        let elem = angular.element('<div class="navbar-absolute-top" nav-controller />');
>
> That's a nice trick :D
>
> That kind of coverage is useless, coverage is just a number after all, and
> unit tests requires isolation to be reliable.
>
> Try to write a test suite for your feature that works independently from
> the rest.
>
> Or leave it as it was and i'll try to cover it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mcasimir/mobile-angular-ui/pull/378/files/ccdae045247c4f5e44c42d1cbd5319eb3fcb252f#r77906493>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADDzgw1gTJQQaYoHcGATnBh5u1VZH_6Oks5qnysbgaJpZM4Jx8nS>
> .
>